src/languageservice/httpClient.ts

```diff
@@ -134,7 +134,7 @@ export default class HttpClient implements IHttpClient {
             });
 
             response.on('error', err => {
-                reject(new PackageError(`Response error: ${err.code || 'NONE'}`, pkg, err));
+                reject(new PackageError(`Response error: ${err.stack}`, pkg, err));
```

Comment:
>stack [](start = 63, length = 5)

we need to verify that the stack isn't logged via telemetry anywhere since this can violate PII requirements.